src/pages/resultsView/pathwayMapper/ResultsViewPathwayMapper.tsx

```diff
@@ -53,6 +53,13 @@ export default class ResultsViewPathwayMapper extends React.Component<
     @observable
     private activeToasts: React.ReactText[];
 
+    @observable
+    private warningMessage: string | null;
+
+    private readonly DEFAULT_MESSAGE = 'Welcome to Pathways tab...';
+    private readonly LOADING_MESSAGE = 'Loading alteration data...';
```

Comment:
Let's define these outside the class (before or after the `interface IResultsViewPathwayMapperProps`) this way:

```suggestion
    const DEFAULT_MESSAGE = 'Welcome to Pathways tab...';
    const LOADING_MESSAGE = 'Loading alteration data...';
```